runtime/proto/corfu_store_metadata.proto

```diff
@@ -45,3 +45,11 @@ message Record {
     google.protobuf.Any payload = 1;
     google.protobuf.Any metadata = 2;
 }
+
+message ProtobufFileName {
```

Comment:
information provided from both these protobufs is already available in TableDescriptors message.

  map<string, google.protobuf.FileDescriptorProto> fileDescriptors = 1;

Why do we need these new protobufs?

Reply:
This is a new type not a protobuf map right?